includes/wikia/api/ArticlesApiController.class.php

```diff
@@ -358,7 +358,7 @@ public function getNew(){
 			$ns = array_unique( $ns );
 		}
 
-		$key = self::getCacheKey( self::NEW_ARTICLES_CACHE_ID, '', [ implode( '-', $ns ) , $minArticleQuality ] );
+		$key = self::getCacheKey( self::NEW_ARTICLES_CACHE_ID, '', [ implode( '-', $ns ) , $minArticleQuality, $limit ] );
```

Comment:
Sure. As we use only two values in most of the cases, I'm fine with the current implementation. Thanks for looking into it :)